src/main/java/seedu/address/ui/BrowserPanel.java

```diff
@@ -70,4 +106,24 @@ private void handlePersonPanelSelectionChangedEvent(PersonPanelSelectionChangedE
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
         loadPersonPage(event.getNewSelection().person);
     }
+
+    @Subscribe
+    private void handleBrowserUrlChangeEvent(BrowserUrlChangeEvent event) throws CommandException {
+        switch (event.getProcessType()) {
+
```

Comment:
Is the indentation here correct? It looks slightly wonky

Reply:
i think this is correct according to travis, initially i did it differently but they gave me an error

Reply:
#believeintravis